pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p /tmp/logdir
+output_dir="/tmp/logdir"
+kubectl get po,svc,pvc --all-namespaces > $output_dir/kubectl-get-po-svc-pvc-all-namespace_output
+IFS=$'\n'
+for i in $(cat $output_dir/kubectl-get-po-svc-pvc-all-namespace_output | grep -i pravega)
+do
+echo $i
+if [[ $i == *"po"* ]];
```

Comment:
Use `*"pod/"*` instead to avoid false positives.

Reply:
@adrianmo I think this search string would be good fit to cover all deployment (GKE + PKS), please let me know.
`if [[ $i == *"po/"* ]] || [[ $i == *"pod/"* ]]; then `